ably.d.ts

```diff
@@ -2028,7 +2035,124 @@ export declare interface PushChannel {
 /**
  * Enables the LiveObjects state to be subscribed to for a channel.
  */
-export declare interface LiveObjects {}
+export declare interface LiveObjects {
+  /**
+   * Retrieves the root {@link LiveMap} object for state on a channel.
+   *
+   * @returns A promise which, upon success, will be fulfilled with a {@link LiveMap} object. Upon failure, the promise will be rejected with an {@link ErrorInfo} object which explains the error.
+   */
+  getRoot(): Promise<LiveMap>;
+}
+
+/**
+ * The `LiveMap` class represents a synchronized key/value storage, similar to a JavaScript Map, where all changes are synchronized across clients in realtime.
```

Comment:
```suggestion
 * The `LiveMap` class represents a synchronized key/value map data structure, similar to a JavaScript Map, where all changes are synchronized across clients in realtime.
```

Reply:
I also assume you would like to remove the `synchronized` word from the `synchronized key/value map data structure` string, the same way you suggested for the counter below

Reply:
```suggestion
 * The `LiveMap` class represents a ey/value map data structure, similar to a JavaScript Map, where all changes are synchronized across clients in realtime.
```